workspace/api/openfdd_bridge/bacnet_poll_worker.py

```
@@ -0,0 +1,47 @@
+    while True:
+        sleep_s = float(os.environ.get("OFDD_POLL_INGEST_INTERVAL_S", "30"))
+        try:
+            path = bacnet_poll_csv()
+            if path.is_file():
+                mtime = path.stat().st_mtime
+                if mtime > _last_mtime:
+                    _last_mtime = mtime
+                    result = ingest_poll_samples_to_feather()
+                    if result.get("ok"):
+                        _log.info("feather ingest: %s", result.get("sites"))
+        except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)